server/mailet/mailetcontainer-camel/src/main/java/org/apache/james/mailetcontainer/impl/JamesMailetContext.java

```diff
@@ -70,6 +70,7 @@
      * A hash table of server attributes These are the MailetContext attributes
      */
     private final Hashtable<String, Object> attributes = new Hashtable<String, Object>();
+    private final static String SMTP_AUTH_USER_ATTRIBUTE_NAME = "org.apache.james.SMTPAuthUser";
```

Comment:
I don't like that header duplication. It's probably a very common attribute for mailets, I suggest we move it to mailet-api then